projects/ccf-shared/src/lib/components/tracking-popup/tracking-popup.component.html

```diff
@@ -0,0 +1,3 @@
+<p>We log usage to improve this service.</p>
+<button mat-button color="basic" (click)="submit(true)">I understand</button>
```

Comment:
Add variations for when they've already opted in/out that makes more sense. For instance: when I've already opted in, you only need to show an Opt Out button. If I'm opted out, only present an Opt In button. Etc.